src/OpenColorIO/OCIOYaml.cpp

```diff
@@ -1739,8 +1779,8 @@ OCIO_NAMESPACE_ENTER
                     if(second.Type() != YAML::NodeType::Map)
                     {
                         std::ostringstream os;
-                        os << "'displays' field needs to be a (name: key) map.";
-                        throw Exception(os.str().c_str());
+                        os << "The value type of 'displays' key needs to be a map.";
+                        throwValueError(node.Tag(), first, os.str());
                     }
```

Comment:
This is nit picky, but some of the throwValueError msg language is inconsistent:

```
305: << ", the value parsing of the key '" << keyName
1718: os << "The value type of 'environment' key needs to be a map.";
1764: os << "The value type of the 'roles' key needs to be a map.";
1782: os << "The value type of 'displays' key needs to be a map.";
```
Perhaps instead:
```
305: << ", the value parsing of the '" << keyName << "' key "
1718: os << "The value type of the 'environment' key needs to be a map.";
1764: os << "The value type of the 'roles' key needs to be a map.";
1782: os << "The value type of the 'displays' key needs to be a map.";
```
Or:
```
305: << ", the value parsing of the key '" << keyName
1718: os << "The value type of the key 'environment' needs to be a map.";
1764: os << "The value type of the key 'roles' needs to be a map.";
1782: os << "The value type of the key 'displays' needs to be a map.";
```
Or any other consistent wording you prefer.

Reply:
Good point. I will fix that.